src/main/java/uk/gov/hmcts/reform/userprofileapi/Application.java

```diff
@@ -24,4 +24,5 @@
     public static void main(final String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
```

Comment:
revert this file,  as there is no functional change.

Reply:
will do, was done just to kick off build.

Reply:
ok fine.